cms/templatetags/cms_tags.py

```diff
@@ -443,3 +443,9 @@ def get_context(self, context):
         context['CMS_TOOLBAR_CONFIG'] = context['request'].toolbar.as_json(context)
         return context
 register.tag(CMSToolbar)
+
+
+def get_by_key(mapping, key):
+    """ Get a value by key from a mapping. """
+    return mapping.get(key, None)
+register.filter(get_by_key)
```

Comment:
If the key name is not a valid django template variable name that lookup cannot be used. In the current case I want to use it for dict ['data-*'] for the widget attribute so we keep a valid html.